components/error_page/error_page.tsx

```diff
@@ -3,24 +3,28 @@
 
 import crypto from 'crypto';
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 import {Link} from 'react-router-dom';
 
 import {ErrorPageTypes, Constants} from 'utils/constants';
 import WarningIcon from 'components/widgets/icons/fa_warning_icon';
 
-import ErrorTitle from './error_title.jsx';
-import ErrorMessage from './error_message.jsx';
+import ErrorTitle from './error_title';
+import ErrorMessage from './error_message';
 
-export default class ErrorPage extends React.PureComponent {
-    static propTypes = {
-        location: PropTypes.object.isRequired,
-        asymmetricSigningPublicKey: PropTypes.string,
-        siteName: PropTypes.string,
-        isGuest: PropTypes.bool,
-    };
+interface ILocation {
+    search: string;
+}
```

Comment:
I wasn't sure about this - any preference for `interface` or `type` or any other suggestions (perhaps about naming) more than welcome!